src/docs/sdk/client-reports.mdx

```diff
@@ -84,7 +84,7 @@ The following discard reasons are currently defined for `discarded_events`:
 - `event_processor`: an event was dropped by an event processor; may also be used for ignored exceptions / errors
 
 In case a reason needs to be added,
-it also has to be added to the allowlist in snuba (`outcomes_processor.py`).
+it also has to be added to the allowlist in [snuba](https://github.com/getsentry/snuba/blob/master/snuba/datasets/outcomes_processor.py#L24-L34).
```

Comment:
```suggestion
it also has to be added to the allowlist in [snuba](https://github.com/getsentry/snuba/blob/4e7cfdddcf7b93eacb762bc74ca2461cec9464e5/snuba/datasets/outcomes_processor.py#L24-L34).
```

Permlinks need the hash otherwise the lines can change.